oauth/v1/types.go

```diff
@@ -37,6 +37,10 @@ type OAuthAccessToken struct {
 
 	// RefreshToken is the value by which this token can be renewed. Can be blank.
 	RefreshToken string `json:"refreshToken,omitempty" protobuf:"bytes,9,opt,name=refreshToken"`
+
+	// InactivityTimeoutSeconds is the seconds from CreationTime after which
+	// this token times out if no activity is detected.
```

Comment:
Probably at minimum add `If a token times out then it can no longer be used and the user will need to request a new access token.`